src/map/utils/zoneutils.cpp

```diff
@@ -109,7 +109,6 @@ namespace zoneutils
             return g_PZoneList.at(zoneId);
         }
 
-        ShowWarning(fmt::format("Invalid zone requested: {}", zoneId));
```

Comment:
This thing is super noisy and is meaningless in a multi-process setup so good riddance

All it means is the zoneId isn't loaded on the current process